padertorch/train/hooks.py

```diff
@@ -816,3 +817,43 @@ def close(self, trainer):
         if self.onset_value is not None:
             module = self.get_module(trainer)
             setattr(module, self.name[-1], self.onset_value)
+
+
+class LRSchedulerHook(TriggeredHook):
+    """
+    A hook that applies a learning rate scheduler from `torch.optim.lr_scheduler`
+    to the training.
+
+    Examples:
+        >>> trainer = pt.Trainer(...)   # doctest: +SKIP
+        >>> trainer.register_hook(LRSchedulerHook(
+        ...     torch.optim.lr_scheduler.StepLR(
+        ...         trainer.optimizer.optimizer, step_size=2, gamma=0.98)
+        ... ))  # doctest: +SKIP
+
+    Note:
+
+        This hook can only be used with learning rate schedulers that
+        don't require metrics.
+
+    """
+    # It is very likely that this check is exclusive to this hook
+    IS_PYTORCH_1_1 = LooseVersion(torch.__version__) >= '1.1.0'
```

Comment:
Can you change this to something like `PYTORCH_ge_1_1`? Do you have an issue, where they mentioned this shift?

Reply:
Yes, and yes it's https://github.com/pytorch/pytorch/pull/7889

Reply:
Thanks, could you add that reference to the issue to the source code?

Reply:
Done!

Reply:
Thanks, I found them directly in the master.